tools/gen.sh

```diff
@@ -15,8 +15,10 @@ qt \
 cp -a google/events/. .
 rm -r google
 
-echo $ROOT
-
 # Run postgen
-cd tools
-npm start
+echo '- Running postgen step'
+(cd tools && npm start)
+
+# Re-format the golang files
+echo "- Re-format golang files"
+gofmt -s -w .
```

Comment:
No Action: Nice addition to the repo!

Good to see this is part of the pipeline now. It would have been better to see this in a separate PR but overall the fmt changes are limited because the existing code quality was not far off the standard.